selfdrive/car/toyota/interface.py

```diff
@@ -9,10 +9,34 @@
 EventName = car.CarEvent.EventName
 
 
+def compute_gb_toyota(accel, speed):
+  return float(accel) / CarControllerParams.ACCEL_SCALE
+
+
+def compute_gb_gas_interceptor(accel, speed):
+  # This converts desired positive acceleration at a speed to gas percentage
+  # It's only accurate up to MIN_ACC_SPEED (19 mph) for now since the function was fitted on data up to that speed
+  # Once we reach that speed, we switch to sending acceleration anyway so this isn't a problem
+  def accel_to_gas(_speed, _accel):  # converts accel to gas using current speed
+    _c1, _c2, _c3, _c4 = [0.04412016647510183, 0.018224465923095633, 0.09983653162564889, 0.08837909527049172]
+    return (_accel * _c1 + (_c4 * (_speed * _c2 + 1))) * (_speed * _c3 + 1)
+
+  if accel > -0.1 and speed <= MIN_ACC_SPEED:  # todo: -0.1 is smooth but in data user was giving gas when a_ego was down to -0.5
```

Comment:
Also I'm not sure what the best way to handle the transition would be. If it was `accel > 0`, 0 accel with the function is some amount of gas, so we don't want a quick hard switch if accel is hovering around a small negative value to 0.

Reply:
Todo: Test what happens if you request a very small amount of negative acceleration while going up a hill under 19 mph. (Ideally) Does the car apply gas to keep us from undershooting the desired acceleration even though we can't apply positive acceleration at that speed? If so, then we can just do accel > 0

Reply:
I'm going to try to use a function I fit on data that gives us the deceleration of the car given a speed under 19 mph, then we check if accel is greater than that to use the converted gas, else use the regular compute_gb function of dividing it by 3.